CompuCell3D/core/CompuCell3D/steppables/PDESolvers/ReactionDiffusionSolverFVM.cpp

```diff
@@ -1554,6 +1680,28 @@ double ReactionDiffusionSolverFV::getFieldDiffusivityInMedium(unsigned int _fiel
 	else { return solver->getCellDiffusivityCoefficient(cell, _fieldIndex); }
 }
 
+double ReactionDiffusionSolverFV::cellInterfaceFlux(unsigned int _fieldIndex, unsigned int _surfaceIndex, CellG *cell, CellG *nCell) {
```

Comment:
Does it mean that we will be able to specify what is going on at the cell interface? If so many people will be happy

Reply:
Correct. It's currently available on the API, and support needs added in the CC3DML.